src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -92,27 +105,17 @@ class SearchContainer extends React.Component {
             />
           </div>
           <div role="search" aria-label="ResearchNow">
-            <div className="nypl-row">
-              <div className="nypl-column-full">
-                <h1 className="nypl-heading">ResearchNow</h1>
-                <div id="tagline">
-                  Search the world&apos;s research collections and more for digital books you can
-                  use right now.
+            {(!searchResults || _isEmpty(searchResults)) && (
+              <div className="nypl-row">
+                <div className="nypl-column-full">
+                  <h1 className="nypl-heading">ResearchNow</h1>
+                  <div id="tagline">Search the world&apos;s research collections and more for digital books you can use right now.</div>
                 </div>
               </div>
-            </div>
+            )}
             <div className="wrapper">
-              <SearchForm
-                searchQuery={searchQuery}
-                searchField={selectedField}
-                history={this.context.history}
-                {...this.boundActions}
-              />
-              <SearchResults
-                results={this.props.searchResults}
-                eReaderUrl={this.props.eReaderUrl}
-                {...this.boundActions}
-              />
+              <SearchForm searchQuery={selectedQuery} searchField={selectedField} history={history} {...this.boundActions} />
+              <SearchResults results={searchResults} eReaderUrl={eReaderUrl} {...this.boundActions} />
```

Comment:
Seems it's much more readable to have these as a multiline instead of inline, but if AirBnB wants it on a single line...